demosplan/DemosPlanCoreBundle/Addon/RpcAddonAssetsLoader.php

```diff
@@ -78,9 +78,10 @@ public function isTransactional(): bool
 
     public function validateRpcRequest(object $rpcRequest): void
     {
-        if (!$this->permissions->hasPermission('area_admin_consultations')) {
-            throw AccessDeniedException::missingPermission('area_admin_consultations');
-        }
+        // TODO: This needs a better validation
+        //if (!$this->permissions->hasPermission('area_admin_consultations')) {
+        //    throw AccessDeniedException::missingPermission('area_admin_consultations');
+        //}
```

Comment:
We should fix this before we merge it, I'm afraid

Reply:
That's not my Baustellenflatterband-comment. 😇  (I'll have a look.)